configs/default/RUSH-BLADE_F4.config

```
@@ -0,0 +1,143 @@
+# Betaflight / STM32F405 (S405) 4.4.0
+
+#define USE_GYRO
+#define USE_GYRO_SPI_MPU6000
+#define USE_ACC
+#define USE_ACC_SPI_MPU6000
+#define USE_GYRO_SPI_ICM42688P
+#define USE_ACC_SPI_ICM42688P
+#define USE_ACCGYRO_BMI270
+#define USE_BARO
+#define USE_BARO_BMP280
+#define USE_BARO_DPS310
+#define USE_FLASH
+#define USE_FLASH_W25Q128FV
+#define USE_MAX7456
+
+board_name  BLADE_F4
+manufacturer_id  RUSH
```

```suggestion
board_name BLADE_F4
manufacturer_id RUSH
```
You have an extra space between the parameter and the value...